tests/integration/pilot/sidecar_api_test.go

```diff
@@ -91,10 +91,10 @@ func validateListenersNoConfig(t *testing.T, response *structpath.Instance) {
 			Select("{.resources[?(@.address.socketAddress.portValue==15001)]}").
 			Equals("virtualOutbound", "{.name}").
 			Equals("0.0.0.0", "{.address.socketAddress.address}").
-			Equals("mixer", "{.filterChains[0].filters[0].name}").
-			Equals("envoy.tcp_proxy", "{.filterChains[0].filters[1].name}").
-			Equals("PassthroughCluster", "{.filterChains[0].filters[1].config.cluster}").
-			Equals("PassthroughCluster", "{.filterChains[0].filters[1].config.stat_prefix}").
+			Equals("mixer", "{.filterChains[1].filters[0].name}").
```

Comment:
shouldn't this be updated to verify that the first filter chain has a condition match for pod ip and a black hole cluster filter?

Reply:
Good point, just added